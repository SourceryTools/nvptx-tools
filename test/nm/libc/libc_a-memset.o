// BEGIN PREAMBLE
.version 3.1
.target sm_35
.address_size 64
// END PREAMBLE
// BEGIN GLOBAL FUNCTION DECL: memset
.visible .func (.param .u64 %value_out) memset (.param .u64 %in_ar0, .param .u32 %in_ar1, .param .u64 %in_ar2);
// BEGIN GLOBAL FUNCTION DEF: memset
.visible .func (.param .u64 %value_out) memset (.param .u64 %in_ar0, .param .u32 %in_ar1, .param .u64 %in_ar2)
{
[...]
}
