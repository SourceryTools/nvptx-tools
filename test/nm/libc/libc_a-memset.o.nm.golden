0000000000000000 T memset
